bundles/org.openhab.binding.warmup/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="warmup"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="my-warmup">
+		<label>My Warmup Account</label>
+		<description>Connection to the https://my.warmup.com site</description>
+		<category>WebService</category>
+		<config-description>
+			<parameter name="username" type="text" required="true">
+				<label>Username</label>
+				<description>Username for my.warmup.com</description>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<label>Password</label>
+				<description>Password for my.warmup.com</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" unit="s" required="true">
+				<label>Refresh Interval</label>
+				<description>Interval in seconds between automatic refreshes</description>
+				<default>300</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="room">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="my-warmup"/>
+		</supported-bridge-type-refs>
+
+		<label>Room</label>
+		<description>Warmup 4iE Device controlling a room</description>
+		<category>RadiatorControl</category>
+
+		<channels>
+			<channel id="currentTemperature" typeId="currentTemperature"/>
+			<channel id="targetTemperature" typeId="targetTemperature"/>
+			<channel id="overrideRemaining" typeId="overrideRemaining"/>
+			<channel id="runMode" typeId="runMode"/>
+		</channels>
+
+		<representation-property>serialNumber</representation-property>
+
+		<config-description>
+			<parameter name="serialNumber" type="text" required="true">
+				<label>Serial Number</label>
+			</parameter>
+			<parameter name="overrideDuration" type="integer" unit="m" required="true">
+				<label>Override Duration</label>
+				<description>Duration of override when target temperature is changed</description>
+				<default>60</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="currentTemperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Current Temperature</label>
+		<description>Current temperature in room, may be air or floor</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="targetTemperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Target Temperature</label>
+		<description>Target temperature currently set on device</description>
+		<category>Heating</category>
+		<state min="5" max="30" step="0.5" readOnly="false" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="overrideRemaining">
```

Comment:
Is there a reason why you didn't document these Channels?

Reply:
No reason, I've added them to the updated readme